custom/sensors/CANEncoder.java

```diff
@@ -144,7 +144,6 @@ public boolean getStoppedSafely() throws InvalidSensorException {
 	@Override
 	public void reset() {
 		super.write(new byte[] {0x72, 0x65, 0x73, 0x65, 0x74, 0x65, 0x6e, 0x63}); // resetenc
```

Comment:
Again, perhaps this byte sequence should be a constant somewhere? (I don't know enough about what it actually is to say one way or the other)